### DISCUSSION TITLE
2024 Cohort 2 - Group 14 Discussion

### DISCUSSION THREAD

**Discussion by sharon-digipres** [Digital-Preservation-Coalition/PythonStudyGroups#40] [General]
This is the discussion thread for those in **Group 14** of the July-December 2024 Cohort

**Comment by nlobraico** [comment]
Hi everyone! This is a test

**Comment by ccllaa** [comment]
Hello everybody, I am Claudia

**Comment by rbfettig** [comment]
Hi! This is a test!

**Comment by laradfs** [comment]
Great to meet you all.  / Lara

**Comment by jedijuli** [comment]
Hi all! Testing this out too. :)
- Julienne

**Comment by AilieOHagan** [comment]
Hi everyone - Ailie here! Great to meet you all today :)

**Comment by llawlis** [comment]
Hi everyone! 

I'm trying to get Bagit.Python running - I've never tried running a Python script before/well never got one to work! 

I've installed Python and Homebrew - I don't know why I can't get the Pip command to work?
<img width="838" alt="Screenshot 2024-07-17 at 4 31 28 PM" src="https://github.com/user-attachments/assets/0d1d8293-0d20-4e54-bfd2-f5996e06b794">

Any ideas? :)

**Reply by llawlis** [reply]
I found this article which was helpful: https://patchbay.tech/blog/2017/12/15/using-bagit-in-2018/

**Reply by llawlis** [reply]
I tried changing the Shell to Bash vs. Zsh as well but have no idea if that worked or not! 

Happy for any advice!

**Reply by AilieOHagan** [reply]
Hi @llawlis I haven't tried this with BagIt yet, but I've only been able to get pip to work for me when I use it in my normal operating systems terminal (I just used the Command Prompt). Not sure if that helps at all - and you might have already gotten it working by now! This sounds like it could be a nice one for us to try out together in the Study Group :)

**Reply by AilieOHagan** [reply]
> I tried changing the Shell to Bash vs. Zsh as well but have no idea if that worked or not!
> 
> Happy for any advice!

Hi @llawlis - I found a handy script you can use if you want to check whether a module is installed or not. Maybe try: 

![Import lib](https://github.com/user-attachments/assets/625abf6b-8955-410a-9fb7-bbf92dfc7fb1)

**Comment by llawlis** [comment]
Thanks @AilieOHagan!!! I will try that and let you know how it goes!

**Comment by AilieOHagan** [comment]
Hi everyone - just sharing a [DigiPres Sandbox](https://www.digipres.org/workbench/tools/sandbox) developed by DPC's Andy Jackson. Could be a useful space for us to test out code before we let it loose on our work stations! The link to Andy's DigiPres Workbench is: https://github.com/digipres/workbench

**Reply by laradfs** [reply]
Thanks for sharing, Ailie.  It's not really clear to me how to use this.  If you have and found it helpful, maybe you can demonstrate at our next meeting.

**Comment by laradfs** [comment]
Hi all, per our meeting today, I was going to try to upload the code I have been working on to our [space here in Github](https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/tree/main/2024%20Cohort%202/Group%2014).  However, when I try to do so, it says uploads are disabled.  Am I missing something?

**Reply by ccllaa** [reply]
Hi Lara, you need to request write access for the code repository of group 14 here: 
https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/discussions

**Comment by ccllaa** [comment]
I am not going to make it today. I just returned from my holidays, and need to catch up with everything. Have a good meeting!

**Comment by ccllaa** [comment]
Hi everybody from Cohort 14!
Again, I cannot make it. The weeks before Xmas are so busy! It was a pleasure to have met you! Even though I spent less time then I wanted for the Python course, I profited from it. I wish you a nice Xmas holiday!
